views/posts/show.hbs

```diff
@@ -4,6 +4,26 @@
       <h1>{{post.title}}</h1>
       {{{post.content}}}
     </div>
+
+    {{#each comments as |comment|}}
+      <div class="card card-block">
+        <p>
+          {{comment.content}}
+        </p>
+      </div>
+    {{/each}}
+    
+    <h5>Add a comment:</h5>
+
+    <form action="/comments" method="POST">
+      <input type="hidden" name="PostId" value={{post.id}} />
```

Comment:
it sends the data along with the post request, thus we have a "PostId" property with the actual id of the post as the value inside req.body object